vitrivr-engine-module-features/src/main/kotlin/org/vitrivr/engine/base/features/external/common/FulltextRetriever.kt

```diff
@@ -1,4 +1,4 @@
-package org.vitrivr.engine.base.features.fulltext
+package org.vitrivr.engine.base.features.external.common
```

Comment:
I do not agree with this move, I guess there could be other use cases than external feature extractions for fulltext.

Reply:
This is accidental, will fix